grammar.g

```diff
@@ -48,6 +49,8 @@ identifier integer_constant float_constant main
 <additive_expression> : <additive_expression> + <multiplicative_expression>
 <additive_expression> : <additive_expression> - <multiplicative_expression>
 <shift_expression> : <additive_expression>
+<shift_expression> : cin >> <additive_expression>
+<shift_expression> : cout << <additive_expression>
```

Comment:
Why only additive expression in cin, cout ?

Reply:
read the entire grammar file

Reply:
Okay then, I am approving.